config/kubernetes/sig-release/teams.yaml

```diff
@@ -0,0 +1,165 @@
+teams:
+  kubernetes-milestone-maintainers:
+    description: Team members can use `/milestone` or `/status` commands on issues/PRs
+    maintainers:
+    - calebamiles # Release / PM
+    - justaugustus # Azure / PM / Release
+    - tpepper # Release
+    - spiffxp # 1.14 RT Lead / Testing
+    members:
+    - abgworrall # GCP
+    - andrewsykim # Cloud Provider
+    - bgrant0607 # Architecture
+    - bradamant3 # Docs
+    - brancz # Instrumentation
+    - bsalamat # Scheduling
+    - carolynvs # Service Catalog
+    - caseydavenport # Network
+    - cblecker # ContribEx
+    - chenopis # Docs
+    - childsb # Storage
+    - csbell # Multicluster
+    - d-nishi # AWS
+    - danielromlein # UI
+    - dcbw # Network
+    - dchen1107 # Node
+    - deads2k # API Machinery
+    - derekwaynecarr # Node
+    - directxman12 # Autoscaling
+    - dklyle # OpenStack
+    - dstrebel # Azure
+    - duglin # Service Catalog
+    - enj # Auth
+    - feiskyer # Azure
+    - fejta # Testing
+    - floreks # UI
+    - foxish # Big Data
+    - frapposelli # VMware
+    - hogepodge # Cloud Provider / OpenStack
+    - idvoretskyi # PM
+    - jagosan # Cloud Provider
+    - jdumars # Architecture
+    - justinsb # AWS
+    - k82cn # Scheduling
+    - khenidak # Azure
+    - kow3ns # Apps
+    - kris-nova # AWS
+    - lavalamp # API Machinery
+    - liyinan926 # Big Data
+    - luxas # Cluster Provider
```

Comment:
typo: s/Cluster Provider/Cluster Lifecycle :P

Reply:
Thanks for catching that, @nikhita!